packages/components/src/components/as-category-widget/as-category-widget.scss

```diff
@@ -0,0 +1,81 @@
+@import '../../../../styles/src/core/_dev';
+
+$bar--height: get-spacing(1);
+
+as-category-widget {
+  --category-widget--description--color: #{$color-type-20};
+  --category-widget--bar--height: #{$bar--height};
+  --category-widget--background-color: #{$color-ui-10};
+  --category-bar--background-color: #{$color-ui-20};
+
+  display: block;
+  background: var(--category-widget--background-color, $color-ui-10);
+
+  @include padding(4);
+
+  .as-category-widget__description,
+  .as-category-widget__count {
+    color: var(--description--color, $color-type-20);
+    color: $color-type-20;
+  }
+
+  .as-category-widget__list {
+    @include padding(0 0 4);
+  }
+
+  .as-category-widget__description + .as-category-widget__list {
+    margin-top: get-spacing(4);
+  }
+
+  .as-category-widget__category-title {
+    position: relative;
+
+    &::after {
+      content: attr(data-value);
```

Comment:
I'm not sure about using the pseudoelement for showing content in the html

Reply:
Yes, I didn't think about accessibility 😞.
And leveraging this situation I think I am going to make an improvement and use an ellipsis to cut overflowed text :)